build.gradle

```diff
@@ -38,6 +38,7 @@ ext {
 
 repositories {
     mavenCentral()
+    mavenLocal()
```

Comment:
I don't agree. I think it will be first or won't be because if you build project via `clean build pTML` gradle publishes artifact as release. 
If we want to write mavenLocal() in the build.gradle script it should be first otherwise we shouldn't be specified